core-backend/sandbox/src/funcs.rs

```diff
@@ -396,8 +396,35 @@ impl<E: Ext + 'static> FuncsHandler<E> {
             .map_err(|_| {
                 ctx.trap = Some("Trapping: unable to send reply message");
                 HostError
-            });
-        result
+            })
+    }
+
+    pub fn reply_wgas(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
+        let mut args = args.iter();
+
+        let payload_ptr = pop_i32(&mut args)?;
+        let payload_len = pop_i32(&mut args)?;
+        let value_ptr = pop_i32(&mut args)?;
+        let gas_limit_ptr = pop_i32(&mut args)?;
```

Comment:
I was already asking for common order of arguments: .. -> payload -> gas limit -> value.

```suggestion
        let gas_limit_ptr = pop_i32(&mut args)?;
        let value_ptr = pop_i32(&mut args)?;
```

Reply:
To follow other api-style